views/components/GraphTable/GraphTable.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+} from '@material-ui/core'
+import { formatSiteData, graphTableColumns } from '../../fe-utils/helpers'
+import { graphTableRowDataBySiteUtil } from './helpers'
+
+const GraphTable = ({ graph }) => {
+  if (!graph) return null
+
+  const tableColumns = graphTableColumns(graph.labels)
+  const [totalsSite, ...sites] = graphTableRowDataBySiteUtil(graph.dataBySite)
```

Comment:
how do we know the `totalsSite` is going to be first? Is that by coincidence? We may need logic in this helper function to ensure that's the case.

Reply:
From your comment above I sorted the new site list and added logic to put Totals last, this way there's no deconstruction and just one map to render the data.
https://github.com/AMP-SCZ/dpdash/pull/193/commits/325e6635a9f09c0b3ce28c6d2a5ab8bfa1369601